app/controllers/UserAnalysis.scala

```diff
@@ -31,11 +31,16 @@ final class UserAnalysis(
       case _ => load("", Standard)
 
   def load(urlFen: String, variant: Variant) = Open:
-    val decodedFen: Option[Fen.Full] = lila.common.String
+    val input: Option[String] = lila.common.String
       .decodeUriPath(urlFen)
       .filter(_.trim.nonEmpty)
       .orElse(get("fen"))
-      .map(Fen.Full.clean)
+    val parsed960: Option[Fen.Full] = input
+      .ifTrue(variant.chess960)
+      .orElse(get("position"))
+      .flatMap(_.toIntOption.flatMap(Chess960.positionToFen))
```

Comment:
and then, if `position` isn't defined, it attempts to convert the FEN into an Int?

Reply:
I was trying to account for `/chess960?poisiton={num}` as well as `/chess960/positionNum` like we allow for `fen`. But I realise that was probably unnecessary